internal/abci/abci.go

```diff
@@ -496,16 +479,30 @@ func (a *AbciApp) InitChain(ctx context.Context, req *abciTypes.RequestInitChain
 
 // ApplySnapshotChunk is on the state sync connection
 func (a *AbciApp) ApplySnapshotChunk(ctx context.Context, req *abciTypes.RequestApplySnapshotChunk) (*abciTypes.ResponseApplySnapshotChunk, error) {
-	refetchChunks, status, err := a.bootstrapper.ApplySnapshotChunk(req.Chunk, req.Index)
+	if a.statesyncer == nil {
+		return nil, errors.New("bootstrapper not set")
```

Comment:
Since I suspect that this will be a common error users hit, could we be more descriptive about what is happening here? Something along the lines of `snapshots are not configured on your local node...` (ideally a more specific diagnosis than that if that is not what is wrong here).